ChustaSoft.Tools.SecureConfig/IServiceProviderExtensions.cs

```diff
@@ -25,9 +24,9 @@ internal static void PerformSettingsDecryption<TSettings>(this IServiceProvider
                 {
                     var writableOptions = GetWritebleSettings<TSettings>(scope, environmentFile, settingsParamName);
 
-                    if (writableOptions.IsAlreadyEncrypted())
+                    if (writableOptions.IsAlreadyEncrypted)
```

Comment:
Renaming this property to `IsCurrentlyEncrypted` could improve readability a (little) bit in the decryption task, as "IsAlready" implies that something can be skipped

Reply:
I like it, what if we directly name it "IsEncrypted" ??

Reply:
Yes! That is better!

Reply:
done! thanks mate